<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPart1</class>
 <widget class="QWidget" name="EditPart1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QTextEdit" name="textEdit_Quest">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}</string>
     </property>
     <property name="text">
      <string>1)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QTextEdit" name="textEdit_Ans1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QRadioButton" name="radioButton_1">
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>151</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
background-color: transparent;
color: black;
}

QRadioButton::indicator::unchecked{ 
background-color: white;
border-radius: 5px;
border: 2px solid black;
}

QRadioButton::indicator::unchecked:disabled{ 
background-color: rgba(110, 110, 110, 255);
border-radius: 5px;
border: 2px solid black;
}

QRadioButton::indicator::checked{ 
background-color: black;
border-radius: 5px;
border: 2px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>242</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
selection-color: black;
}

QComboBox::drop-down
{
border-radius: 4px;
border: 2px solid rgb(0, 135, 0); 
width: 15px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(202, 205, 0, 255), stop:1 rgba(255, 255, 136, 255));
 }

QComboBox::drop-down:pressed 
{
border-radius: 4px;
border: 2px solid rgb(0, 135, 0); 
width: 15px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 136, 255), stop:1 rgba(202, 205, 0, 255));
 }

QComboBox QAbstractItemView {
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(202, 205, 0, 255), stop:1 rgba(255, 255, 136, 255));
selection-color: black;
}

</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}</string>
     </property>
     <property name="text">
      <string>2)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QTextEdit" name="textEdit_Ans2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>151</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
background-color: transparent;
color: black;
}

QRadioButton::indicator::unchecked{ 
background-color: white;
border-radius: 5px;
border: 2px solid black;
}

QRadioButton::indicator::unchecked:disabled{ 
background-color: rgba(110, 110, 110, 255);
border-radius: 5px;
border: 2px solid black;
}

QRadioButton::indicator::checked{ 
background-color: black;
border-radius: 5px;
border: 2px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>298</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}

QLabel:disabled {
border-radius: 7px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
     </property>
     <property name="text">
      <string>3)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QTextEdit" name="textEdit_Ans3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}

QTextEdit:disabled {
border-radius: 7px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QRadioButton" name="radioButton_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>151</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
background-color: transparent;
color: black;
}

QRadioButton::indicator::unchecked{ 
background-color: white;
border-radius: 5px;
border: 2px solid black;
}

QRadioButton::indicator::unchecked:disabled{ 
background-color: rgba(110, 110, 110, 255);
border-radius: 5px;
border: 2px solid black;
}

QRadioButton::indicator::checked{ 
background-color: black;
border-radius: 5px;
border: 2px solid black;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}

QLabel:disabled {
border-radius: 7px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
     </property>
     <property name="text">
      <string>4)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QTextEdit" name="textEdit_Ans4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}

QTextEdit:disabled {
border-radius: 7px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QRadioButton" name="radioButton_4">
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>16</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>151</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
background-color: transparent;
color: black;
}

QRadioButton::indicator::unchecked{ 
background-color: white;
border-radius: 5px;
border: 2px solid black;
}

QRadioButton::indicator::unchecked:disabled{ 
background-color: rgba(110, 110, 110, 255);
border-radius: 5px;
border: 2px solid black;
}

QRadioButton::indicator::checked{ 
background-color: black;
border-radius: 5px;
border: 2px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
